pkg/services/ngalert/sender/sender.go

```diff
@@ -50,6 +50,7 @@ func NewExternalAlertmanagerSender() (*ExternalAlertmanager, error) {
 		sdCancel: sdCancel,
 	}
 
+	s.logger.Info("Initiating communication with a group of external Alertmanagers")
```

Comment:
1. I think it would be great to log the orgID in this context.
2. Please change the logger name to `ngalert.sender.externalAlertmanager` (above)

Reply:
The sender doesn't know its own org ID from this context. I don't really feel great about injecting more data/crossing boundaries purely for log purposes. There's a log statement one layer out contains the org ID

Reply:
Fixed logger name

Reply:
orgID is already in every aspect of the Grafana code. So, I do not think this crosses any boundaries. As alternative, it can accept logger from upstream instead of building it on its own.

Reply:
I actually ended up moving this log message entirely. `NewExternalAlertmanagerSender` can never actually return any errors nor does it start anything, it just wires up some structs. The whole error checking bit here is all dead code. The log doesn't feel relevant here any more.

We still add context, just later on in `ApplyConfig` which is the point at which the sender gets bound to an org.